politeiawww/www.go

```diff
@@ -746,13 +746,12 @@ func _main() error {
 		if err != nil {
 			return err
 		}
-
-		p.initEventManagerPi()
+		// Setup event manager
 		p.setupEventListenersPi()
 
 	case cmsWWWMode:
 		// Setup event manager
-		p.initCMSEventManager()
+		p.setupEventListenersCms()
```

Comment:
Initialisms should be consitently cased `CMS`.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms